index.html

```diff
@@ -152,15 +152,15 @@
       <h2>Use Cases</h2>
       <p>
         JF2 has evolved as a result of a variety of use-cases for different implementations
-        exploring ways to simplify their existing use of canonical parsed microformats2 JSON 
+        exploring ways to simplify their existing use of canonical parsed Microformats-2 JSON
```

Comment:
Also, to clarify what I've done, normative references still say "[microformats2]", references to microformats in general say "microformats", and spcific references to the mf2 spec say "Microformats-2".